### PR TITLE
fix(newKernel): Set kernel directory to ~

### LINKED ISSUE #707: Default launch to ~ when not opened at terminal
On OS X, if I open the latest release from Finder, the current working directory is `/`. Starting out in `~` would be much better for an intro experience.

### PR DESCRIPTION
**Pull request by captainsafia**
This sets the working directory on a new kernel to the home directory. We're now passing the home dir directly to the kernel launcher instead of making it rely on `process.cwd()`.

Closes #707.

### CODE REVIEW

**Comment by rgbkrk** [comment]
Awesome, this is exactly what we needed for a fresh notebook.

**Comment by captainsafia** [comment]
Ugh. What is it with this test failure?!!? Let me look into it.

File: test/renderer/menu-spec.js [hunk 1]
```diff
@@ -248,7 +248,7 @@ describe('menu', () => {
     expect(store.dispatch.firstCall).to.be.calledWith({
       type: 'NEW_NOTEBOOK',
       kernelSpecName: 'perl',
-      cwd: process.cwd(),
+      cwd: require('home-dir')(),
```
Comment (rgbkrk) [inline test/renderer/menu-spec.js:251]: 👍

File: src/notebook/menu.js [hunk 2]
```diff
@@ -221,7 +223,7 @@ export function dispatchLoad(store, event, filename) {
 }
 
 export function dispatchNewNotebook(store, event, kernelSpecName) {
-  store.dispatch(newNotebook(kernelSpecName, remote.app.getPath('home')));
+  store.dispatch(newNotebook(kernelSpecName, home()));
```
Comment (rgbkrk) [inline src/notebook/menu.js:226]: 👍

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
Wish the spammy collapsed if it's the same message.

**Pull request merged**